dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.31/V2_31_23_Upgrade_sequential_nuber_counter_key.sql

```diff
@@ -0,0 +1 @@
+UPDATE sequentialnumbercounter SET key = substring(key, 12, (length(key)-12)) WHERE key LIKE 'SEQUENTIAL(%)';
```

Comment:
Hey, is there also more advanced patterns that we need to support, where SEQUENTIAL() is only part of the pattern?

Reply:
So, had a call with Stian. We figured out that the above fix os OK. An example pattern of BETA-SEQUENTIAL(###) would be stored differently in 2.30, 2.31, 2.32 and beyond.
For 2.30, the sequential part of a pattern is always stored as SEQUENTIAL(###) in the counter table.
For 2.31, the sequential part of a pattern is always stored as ### in the counter table.
For 2.32 and beyond, the entire pattern is stored in the counter table: BETA-###

This PR only takes care of upgrades between 2.30 and 2.31. We will make another PR for upgrades to 2.32 and beyond.